.gitignore

```diff
@@ -6,4 +6,3 @@ yarn-error.log
 .stylelintcache
 .npmrc
 public
-dist
```

Comment:
we're building `dist` after merge to master, to avoid conflicts with other branches. Can we revert this change?